### ISSUE TITLE
Add a delay before automatic disconnect for the A/C disconnect

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#191]
```
The automatic launch (Lancement Auto) is great 
but for example if you disconnect from the power the time to switch to another 
plug, you have a disconnection / reconnection. 

It would be great to have a "time before disconnect" on this screen 0 / 1 
minute / 5 minutes 
```

Original issue reported on code.google.com by `lpenou.tstXX@mpo.li` on 21 Jun 2011 at 11:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 21 Jun 2011 at 8:50
- Changed title: **Add a delay before automatic disconnect for the A/C disconnect**
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 30 Nov 2011 at 10:54
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 5 Dec 2011 at 12:36
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Sorry, translation is missing.
```

Original comment by `fschm...@gmail.com` on 5 Dec 2011 at 12:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 5 Dec 2011 at 12:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 27 Dec 2011 at 11:10
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you :)
```

Original comment by `lpenou@m4p.li` on 28 Dec 2011 at 1:19

**Comment by GoogleCodeExporter** [comment]
```
Fixed and deployed with GTalkSMS 3.4
```

Original comment by `Florent....@gmail.com` on 7 Jan 2012 at 3:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Great It was working fine in the 3.3.3beta 
Hope to install the 3.4 soon 
```

Original comment by `lpenou@m4p.li` on 7 Jan 2012 at 3:41
